.github/workflows/ci-core.yml

```diff
@@ -1,7 +1,8 @@
 name: CI Core
+run-name: CI Core ${{ inputs.distinct_run_name && inputs.distinct_run_name || '' }}
 
 concurrency:
-  group: ${{ github.workflow }}-${{ github.ref }}
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ inputs.distinct_run_name }}
```

Comment:
Added this here so if chainlink-evm pushes two PRs that run these tests around the same time in the future they don't collide with the last one in winning.